src/System Application/Test/Data Classification/src/DataClassificationMgtTests.Codeunit.al

```diff
@@ -104,7 +101,7 @@ codeunit 135150 "Data Classification Mgt. Tests"
         DataSensitivity.DeleteAll();
 
         // [GIVEN] A table number, a field number and a data sensitivity
-        TableNo := 18;
+        TableNo := Database::"Data Sensitivity";
```

Comment:
why the change? before it was using the Customer table, now Data Sensitivity... what's the diff?

Reply:
Resolving so we use the comment below for the same discussion.